rcl/src/rcl/service.c

```diff
@@ -250,6 +255,8 @@ rcl_take_request(
     RCL_SET_ERROR_MSG(rmw_get_error_string_safe(), options->allocator);
     return RCL_RET_ERROR;
   }
+  RCUTILS_LOG_DEBUG_NAMED(
+    ROS_PACKAGE_NAME, "Service take request succeeded: %s", taken ? "true" : "false")
```

Comment:
We could add a debug print to `rcl_send_response` as well (https://github.com/ros2/rcl/pull/187/files#diff-777efe6fef16afbd8f9a9877ffba1138R282)?

Reply:
done in 7c7e11762345e5e16d20367337c785759af789d0, and added for publish/take in 88b8bee2b4665cc5ed9b9846f6b17dc69aaf4ebd